web/vtctld2/src/app/dashboard/shard.component.ts

```diff
@@ -72,20 +65,55 @@ export class ShardComponent implements OnInit, OnDestroy {
     });
   }
 
+  getActions() {
+    let result: MenuItem[] = [
+      {label: 'Status', items: [
+        {label: 'Validate', command: (event) => {this.openValidateShardDialog(); }},
+        {label: 'Validate Version', command: (event) => {this.openValidateVerShardDialog(); }},
+        {label: 'Shard Replication Positions', command: (event) => {this.openShardReplicationPosDialog(); }},
+      ]},
+      {label: 'Change', items: [
+        {label: 'Delete', command: (event) => {this.openDeleteShardDialog(); }},
+        {label: 'Tablet Externally Reparented', command: (event) => {this.openTabExtRepDialog(); }},
```

Comment:
I guess I don't have a strong opinion on where these commands should be. My concern is just that a menu item saying "Tablet ... Reparented" seem to suggest that the tablet I'm selecting it on is reparented. And having such menu item on a shard is confusing. "Externally Reparent Tablet" while not ideal doesn't seem to have such strong suggestion that it should be selected on a tablet.